src/fontra/views/editor/panel-selection-info.js

```diff
@@ -409,6 +428,43 @@ export default class SelectionInfoPanel extends Panel {
     }
   }
 
+  async _toggleGlyphLock(varGlyph) {
+    if (varGlyph.customData["fontra.glyph.locked"]) {
+      const result = await dialog(
+        `Are you sure you want to unlock glyph ${varGlyph.name}?`,
+        "",
+        [
+          { title: "Cancel", isCancelButton: true },
+          { title: "Yes", isDefaultButton: true, resultValue: "ok" },
+        ]
+      );
+
+      if (!result) {
+        // User cancelled
+        return;
+      }
+    }
+
+    await this.sceneController.editGlyphAndRecordChanges(
+      (glyph) => {
+        this.sceneController.selection = new Set();
```

Comment:
Why do you reset the selection? That is not needed. Selection and copying is (and should) still possible in a locked glyph.